src/test/unit/com/leverx/learningmanagementsystem/AbstractTest.java

```diff
@@ -0,0 +1,9 @@
+package com.leverx.learningmanagementsystem;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public abstract class AbstractTest {
```

Comment:
for unit tests there is no need to create common abstract class, it's only for integration tests to avoid recreation of context for each test